vanilla-tests/pom.xml

```diff
@@ -18,12 +18,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version>
+      <version>1.4.0</version>
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.1.0</version>
+      <version>1.4.0</version>
```

Comment:
Just using azure-core-http-netty as dependency would transitively bring in azure-core as well. Also, we have 1.5.0 as the latest netty version.

Reply:
Ideally, netty should be in test scope.